python/pvault_getting_started.py

```diff
@@ -11,193 +11,240 @@
 AUTHORIZATION_HEADER = "Authorization"
 UNSAFE_OPTION = "unsafe"
 PVAULT_AUTHENTICATION = "pvaultauth"
+PVAULT_PORT = 8123
+PVAULT_ADDRESS = f'http://localhost:{PVAULT_PORT}'
 
-PVAULT_ADDRESS = 'http://localhost:8123'
 
-
-# Verify the DB is empty. For safety reasons, don't work on non empty DB
-def check_clear(client):
+def check_db_is_clear(client):
+    # Verify the DB is empty. For safety reasons, don't work on non empty DB
     collections_manager = collections_api.CollectionsApi(client)
 
     all_collections = list(collections_manager.list_collections())
-    assert len(collections_manager.list_collections()) == 0, \
+    assert len(all_collections) == 0, \
         "Bailing out due to existence of collections from previous runs. Please clear or recreate the Vault from scratch."
 
 
-def main():
+def create_api_client():
     # Create configuration, bearer auth and client API
     config = configuration.Configuration(host=PVAULT_ADDRESS)
     config.api_key = {AUTHORIZATION_HEADER: PVAULT_AUTHENTICATION}
     config.api_key_prefix = {AUTHORIZATION_HEADER: "Bearer"}
     client = api_client.ApiClient(config, AUTHORIZATION_HEADER,
                                   config.get_api_key_with_prefix(AUTHORIZATION_HEADER))
 
-    # Create managers - each one responsible for different actions
+    return client
+
+
+def create_api_managers(client):
```

Comment:
why do you cal it, managers?

Reply:
you can combine this func and create_api_client()

Reply:
its the same as it called before (but not in a function)

Reply:
if you combine them you can call it create_api_clients():